%!TEX root = ../proteoform_suite_manual.tex
%---------------------------------------------------------------------
%	TOPDOWN SEARCH
%---------------------------------------------------------------------

\section{Top-down Search}

Results from a top-down MS/MS search can be used in Proteoform Suite. These top-down proteoform identifications improve intact-mass analysis and proteoform family construction. Additionally, top-down results can be integrated with bottom-up peptide results. Top-down results are loaded on the Load Results page (see \textbf{Load Results: Standard}) under Top-Down Hit Results. This section describes how to obtain top-down results to import into Proteoform Suite.

\subsection{TDPortal}
TDPortal is a high-throughput global proteome analysis software for top-down data\supercite{LeDuc2004,Zamdborg2007,Toby2019} available through the National Resource for Translational and Developmental Proteomics.
\begin{itemize}
\item Request for access to TDPortal: \url{http://nrtdp.northwestern.edu/tdportal-request/}
\item Download TDViewer to access results: \url{http://topdownviewer.northwestern.edu/}
\item Under Reports tab in TDViewer, export Hit Report
\item The resulting .xlsx file is used for Top-Down Hit Results in the Standard analysis on the Load Results page
\end{itemize}

\subsection{MetaMorpheus}
MetaMorpheus is an MS/MS search software program for both bottom-up and top-down analysis.\supercite{Solntsev2018,Schaffer2019}
\begin{itemize}
\item On the Load Results page, select MetaMorpheus Top-Down Search under Choose Analysis (top left)
\item Load .raw or .mzML files in the table with the Load Data drop down menu set to Spectra Files
\item Load an .xml or .fasta database in the table with the Load Data drop down menu set to Protein Databases
\item Set Parameters:
\begin{itemize}
\item Precursor Mass Tolerance: the difference in mass between the observed precursor and the theoretical proteoform
\item Product Mass Tolerance: the difference in mass between the product ions generated by fragmentation and the theoretical proteoform's theoretical fragmentation spectra
\item Fixed Carbamidomethyl Mod:  fixed modifications are applied to EVERY amino acid in the database specified in the list. Check this box for protein samples that have been reduced and alkylated with iodoacetamide
\item Dissociation Type: the dissociation type used to fragment intact proteoforms and produce product ions for the tandem mass spectra
\end{itemize}
\item To begin the search, hit the MetaMorppheus Top-Down Search button under Start Analysis (bottom right)
\item For more advanced parameter options, you can also run the GUI or command line version of MetaMorpheus, available at \url{https://github.com/smith-chem-wisc/MetaMorpheus/releases}
\item The resulting AllPSMs.psmtsv file is used for Top-Down Hit Results in the Standard analysis on the Load Results page
\end{itemize}
